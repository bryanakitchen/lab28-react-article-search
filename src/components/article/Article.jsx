import React from 'react';

export default function Article() {
  return (
    <div>
            Hello from the Article component.
    </div>
  );
}
