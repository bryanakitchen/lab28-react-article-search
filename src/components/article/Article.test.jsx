import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './Article';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('renders Article', () => {
    const { asFragment } = render(<Article
      title="Happy Trees"
      author="Bob Ross"
      description="this article talks about drawing happy trees"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
