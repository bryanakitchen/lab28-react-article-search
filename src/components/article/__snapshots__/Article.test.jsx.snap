// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <h2>
    Happy Trees
  </h2>
  <h3>
    Bob Ross
  </h3>
  <p>
    this article talks about drawing happy trees
  </p>
</DocumentFragment>
`;
